modules/raft/src/main/java/org/apache/ignite/internal/raft/storage/impl/DefaultLogStorageFactory.java

```diff
@@ -360,6 +369,49 @@ private static ColumnFamilyOptions createColumnFamilyOptions() {
         return opts;
     }
 
+    @Override
+    public Set<String> raftNodeStorageIdsOnDisk(GroupIdFastForward fastForward) {
```

Comment:
Do we even need "fastForward"?
Can't we go through the meta and get all the groups?